tests/suites/test_suite_psa_crypto.function

```diff
@@ -1960,6 +1961,14 @@ void hash_operation_init( )
 
     memset( &zero, 0, sizeof( zero ) );
 
+    /* A default hash operation should not be usable. */
```

Comment:
What does “default hash operation” mean? I understand from context, but this is the code explaining the comment whereas it should be the other way round. Suggestion: “freshly-initialized”

Reply:
I've also seen us use "blank". I'll change to freshly-initialized.